src/server/pkg/hashtree/hashtree_test.go

```diff
@@ -541,7 +541,7 @@ func TestWalk(t *testing.T) {
 		"/dir":     true,
 		"/dir/bar": true,
 	}
-	require.NoError(t, tree.Walk(func(path string, node *NodeProto) error {
+	require.NoError(t, tree.Walk("", func(path string, node *NodeProto) error {
```

Comment:
Very minor suggestion: I think `Walk("/" func())` might be slightly easier to read (as in, `"/"` suggests that the argument is a path and you're starting at the beginning).  Totally your call on whether it's worth changing, though

Reply:
That seems worthwhile, I'll make the change.